cmd/es_health.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+var (
+	esHealthCmd = &cobra.Command{
+		Use:   "health",
+		Short: "Test ElasticSearch health status",
+		Long:  "Run this command to test health status of ElasticSearch",
+		Run: func(c *cobra.Command, args []string) {
+			res, err := es7Client.Cat.Health()
+			if err != nil {
+				fmt.Println(err)
+			}
+			fmt.Println(res)
+		},
+	}
```

Comment:
Can you post a screenshot of the result when running the command `health`?

Reply:
![health](https://user-images.githubusercontent.com/35046106/59070806-051d5080-8871-11e9-9f44-061ef868a449.PNG)